Description: Don't download test data files; supplied via tarball
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2019-10-14
Forwarded: no


--- a/tests/utils/CMakeLists.txt
+++ b/tests/utils/CMakeLists.txt
@@ -58,7 +58,7 @@ ecbuild_add_test( TARGET      eckit_test
 
 ecbuild_add_test( TARGET      eckit_test_utils_compression_performance
                   CONDITION   HAVE_EXTRA_TESTS
-                  TEST_DEPENDS get_eckit_test_data
+#                  TEST_DEPENDS get_eckit_test_data
                   SOURCES     compression-performance.cc
                   LIBS        eckit )
 
--- a/tests/geo/CMakeLists.txt
+++ b/tests/geo/CMakeLists.txt
@@ -41,27 +41,27 @@ foreach(_test
         LIBS        eckit_geo )
 endforeach()
 
-if(eckit_HAVE_GEO_GRID_ORCA)
-    set(URL "https://get.ecmwf.int/repository/atlas/grids/orca/v0/ORCA2_T.atlas")
-    set(FILE "${CACHE_PATH}/eckit/geo/grid/orca/d5bde4f52ff3a9bea5629cd9ac514410.atlas")
-    set(MD5 "f279b48c171409f46bfd27dff98d454a")
+# if(eckit_HAVE_GEO_GRID_ORCA)
+#     set(URL "https://get.ecmwf.int/repository/atlas/grids/orca/v0/ORCA2_T.atlas")
+#     set(FILE "${CACHE_PATH}/eckit/geo/grid/orca/d5bde4f52ff3a9bea5629cd9ac514410.atlas")
+#     set(MD5 "f279b48c171409f46bfd27dff98d454a")
 
-    file(DOWNLOAD ${URL} ${FILE} EXPECTED_MD5 ${MD5} TIMEOUT 30 STATUS DOWNLOAD_STATUS)
+#     file(DOWNLOAD ${URL} ${FILE} EXPECTED_MD5 ${MD5} TIMEOUT 30 STATUS DOWNLOAD_STATUS)
 
-    list(GET DOWNLOAD_STATUS 0 STATUS_CODE)
-    if(${STATUS_CODE} EQUAL 0)
-        message(STATUS "File downloaded: '${FILE}'")
-    else()
-        list(GET DOWNLOAD_STATUS 1 STATUS_MESSAGE)
-        message(FATAL_ERROR "File download failed: ${STATUS_MESSAGE}, file: '${FILE}'")
-    endif()
+#     list(GET DOWNLOAD_STATUS 0 STATUS_CODE)
+#     if(${STATUS_CODE} EQUAL 0)
+#         message(STATUS "File downloaded: '${FILE}'")
+#     else()
+#         list(GET DOWNLOAD_STATUS 1 STATUS_MESSAGE)
+#         message(FATAL_ERROR "File download failed: ${STATUS_MESSAGE}, file: '${FILE}'")
+#     endif()
 
-    ecbuild_add_test(
-        TARGET      eckit_test_geo_grid_orca
-        SOURCES     grid_orca.cc
-        LIBS        eckit_geo
-        ENVIRONMENT "ECKIT_GEO_CACHE_PATH=${CACHE_PATH}" )
-endif()
+#     ecbuild_add_test(
+#         TARGET      eckit_test_geo_grid_orca
+#         SOURCES     grid_orca.cc
+#         LIBS        eckit_geo
+#         ENVIRONMENT "ECKIT_GEO_CACHE_PATH=${CACHE_PATH}" )
+# endif()
 
 ecbuild_add_test(
     TARGET  eckit_test_geo_tool_grid_spec_1_1
